### ISSUE TITLE
Impossible to filter JSON and return the original JSON?

### ISSUE THREAD

**Issue by adamsiemion** [json-path/JsonPath#507]
Want I am trying to achieve is: extract part of the provided JSON and return the result as `String`. 

It seems all the methods taking a json path (`read*`) return an Object. While the only method returning JSON as String (`jsonString()` is available in `DocumentContext`, which unfortunately does not provide a method to filter JSON. Therefore I am using this code:

     JsonPath.parse(JsonPath.read(stringWithJson, jsonPath)).jsonString();

is there a more efficient way?

**Comment by hf-kklein** [comment]
> is there a more efficient way?

I don't think so. But you're not obliged to use JsonPath.parse(...).jsonString(); to convert your Java object to a json string. You could use any serializer.
